project_sequence/models/project_project.py

```diff
@@ -72,7 +72,7 @@ def create(self, vals_list):
         # It is important to set sequence_code before calling super() because
         # other modules such as hr_timesheet expect the name to always have a value
         for vals in vals_list:
-            if "sequence_code" not in vals:
+            if vals.get("sequence_code", False):
```

Comment:
That's the opposite of the current behavior with no other check.
IMHO, the code should be like this:

```suggestion
           sequence_code = vals.get("sequence_code", False)
            if not sequence_code:
```

As in views, sequence_code is present but with a "" value.

@joscanog

Reply:
Thanks @rousseldenis